app/config/auth.js

```diff
@@ -9,7 +9,7 @@ const schema = Joi.object({
 })
 
 const config = {
-  enabled: process.env.AADAR_ENABLED,
+  enabled: process.env.AADAR_ENABLED === 'true',
```

Comment:
Do we want this in?

Reply:
I think we ought to? Or do you think it might muddy the waters?